README.md

```diff
@@ -80,6 +80,19 @@ how we're organizing branches in our repository in the
 
 Say hello on our [Slack][4] channel: `#mobile`.
 
+## Alternative Build Instructions ##
+
+WordPress-Android can be compiled with [Buck][8], an alternative to Gradle,
+that makes the build process much faster. Due to actual BUck limitation, you
+need to bootstrap the project by running the following command:
+
+    $ ./tools/fetch_buck_dependencies.py extlibs
+
+This command will fetch all dependencies (`.aar` and `.jar`) needed to build
+the project. Then you can run buck to build the apk:
+
+    $ buck build wpandroid
+
```

Comment:
we should add the `buck install --run wpandroid` tip here as well for running it to the device/emu